### ISSUE TITLE
Info::is_complete() - does it only wait on Version to be complete?

### ISSUE THREAD

**Issue by hamishwillee** [dronecore/DroneCore#132]
Docs say:
```
bool dronecore::Info::is_complete() const
Tests if this Version object is fully populated from hardware.
```
But I suspect that `Product` is also waited on.

**Comment by julianoes** [comment]
Yes, product is from the same mavlink message, so the docs are wrong.

**Comment by hamishwillee** [comment]
@julianoes Hope this is the right way to do update - just merge into develop based on develop.

FYI When #135 is merged, also need to update docs to build on top of https://github.com/dronecore/docs/pull/47

**Comment by julianoes** [comment]
Yes, that is perfect, thanks @hamishwillee.
